### PR TITLE
Relationship Arrows Are Now Visualized

### PR DESCRIPTION
**Pull request by AaronWB**
Created a Arrow Manager and Designer Class for managing how relationships are visualized and modified.

Works from Load, and from switching to GUI to CLI.

Different "Heads" to the baseline for Composition/Aggregation and Realization/Generalization relationship types.

Modified the populate class to load the relationships.

### CODE REVIEW

**Review by SergeiAshikhmin — APPROVED** [review]
Looks good to me.

File: UML editor/src/main/java/codecain/GraphicalUserInterface/Model/ArrowManager.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+ */
+public class ArrowManager {
+    private final Pane arrowContainer;
+    private final Map<Relationship, Polygon> arrowMap;
+    private final ArrowDesigner arrowDesigner;
+
+    /**
+     * Constructs an ArrowManager with the specified container and designer.
+     *
+     * @param arrowContainer the pane to which arrows will be added.
```
Comment (colbycrutcher) [inline UML editor/src/main/java/codecain/GraphicalUserInterface/Model/ArrowManager.java:26]: adding visuals, they look good

File: UML editor/src/main/java/codecain/GraphicalUserInterface/View/ArrowDesigner.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+     * TODO: Fix the Diamond being a little bit off
+     *
+     * @param startX the x-coordinate of the start point.
+     * @param startY the y-coordinate of the start point.
+     * @param endX   the x-coordinate of the end point.
+     * @param endY   the y-coordinate of the end point.
+     * @param type   the relationship type.
+     * @return the created polygon representing the arrow.
+     */
+    public Polygon createArrow(double startX, double startY, double endX, double endY, RelationshipType type) {
```
Comment (colbycrutcher) [inline UML editor/src/main/java/codecain/GraphicalUserInterface/View/ArrowDesigner.java:36]: arrows look very nice!

**Review by colbycrutcher — APPROVED** [review]
Overall looks good, from what I see when I run the code

**Pull request merged**